pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -5,6 +5,7 @@
   lcms2,
   installShellFiles,
   dbus,
+  darwin,
```

Comment:
Most packages have UserNotifications as input, which defaults to null, and in the all-packages.nix the test if available is done. Any reason to not do that here?

Reply:
`darwin.apple_sdk.frameworks.UserNotifications` does not exit in sdk 10, and x86_64-darwin use sdk 10.